doc/lua_api.txt

```diff
@@ -1673,17 +1673,37 @@ these formats.
 ### Serialized
 
 This is called "stackstring" or "itemstring". It is a simple string with
-1-3 components: the full item identifier, an optional amount and an optional
-wear value. Syntax:
+1-4 components:
 
-    <identifier> [<amount>[ <wear>]]
+1. Full item identifier ("item name")
+2. Optional amount
+3. Optional wear value
+4. Optional item metadata
+
+Syntax:
+
+    <identifier> [<amount>[ <wear>[ <metadata>]]]
 
 Examples:
 
 * `"default:apple"`: 1 apple
 * `"default:dirt 5"`: 5 dirt
 * `"default:pick_stone"`: a new stone pickaxe
 * `"default:pick_wood 1 21323"`: a wooden pickaxe, ca. 1/3 worn out
+* `[[default:pick_wood 1 21323 "\u0001description\u0002My worn out pick\u0003"]]`:
+  * a wooden pickaxe from the `default` mod,
+  * count of 1, ca. 1/3 worn out (it's a tool),
```

Comment:
How do you get from `21323` to `ca 1/3 worn out`?

Reply:
> How do you get from `21323` to `ca 1/3 worn out`?

Just above this:

> Amount and wear
> ---------------
>
> All item stacks have an amount between 0 and 65535. It is 1 by
> default. Tool item stacks can not have an amount greater than 1.
>
> Tools use a wear (damage) value ranging from 0 to 65535. The
> value 0 is the default and is used for unworn tools. The values
> 1 to 65535 are used for worn tools, where a higher value stands for
> a higher wear. Non-tools technically also have a wear property,
> but it is always 0. There is also a special 'toolrepair' crafting
> recipe that is only available to tools.